.github/workflows/scala.yml

```diff
@@ -54,10 +54,11 @@ jobs:
     - name: Install Z3
       run: sudo apt-get update -y; sudo apt-get install -y z3
     
-    - name: Set up JDK 1.8
+    - name: Set up JDK 15
```

Comment:
Is there a particular reason for choosing version 15 instead of 11 (LTS)?

Reply:
Fábio said that version 11 was more problematic. I think we kind of agreed in the end to upgrade to Java 15 and then again to the next LTS that is coming out later this year. That should be a smaller leap.